18 support multiple nrf cloud accounts (#116)
hello-nrfcloud/backend: features/MultipleNRFCloudAccounts.feature.md

# Multiple nRF Cloud accounts

> After connecting to the websocket with a device fingerprint under specific
> account, the device shadow is returned

> ## Verify a exeger device sends shadow data to nRF Cloud, then there is shadow request with associated api key

Reviewer: ```suggestion
## Verify a ACME device sends shadow data to nRF Cloud and there is a shadow request with the associated API key
```

                  }
                ]
              },
              "networkInfo": {
                "currentBand": {
                  "timestamp": 1682072423
                },
                "networkMode": {
                  "timestamp": 1682072423
                },
                "rsrp": {
                  "timestamp": 1682072423
                },
                "areaCode": {
                  "timestamp": 1682072423
                },
                "mccmnc": {
                  "timestamp": 1682072423
                },
                "cellID": {
                  "timestamp": 1682072423
                },
                "ipAddress": {
                  "timestamp": 1682072423
                },
                "eest": {
                  "timestamp": 1682072423
                }
              }
            }
          }
        }
      }
    }
  ],
  "total": 1,
  "pageNextToken": "4bb1f9ab35bd"
}
```

When I connect to the websocket using fingerprint `${fingerprint}`

<!-- @retry:tries=5,initialDelay=5000,delayFactor=1 -->

Soon there is a device shadow request for `${fingerprint_deviceId}` with API key
of `exeger` account

> ## Verify a nodic device sends shadow data to nRF Cloud, then there is shadow request with associated api key

Reviewer: ```suggestion
## Verify a Nordic device sends shadow data to nRF Cloud, then there is a shadow request with associated API key
```
